app/Platform/RouteServiceProvider.php

```diff
@@ -72,6 +73,7 @@ protected function mapWebRoutes(): void
             // Route::resource('leaderboard', LeaderboardController::class)->only('show');
 
             // Route::get('user/{user}/history', [PlayerHistoryController::class, 'show'])->name('user.history');
+            Route::get('user/{user}/dev/feed', DeveloperFeedController::class)->name('developer.feed');
```

Comment:
I think we're going with `/developer/feed`: https://github.com/RetroAchievements/RAWeb/pull/1998#discussion_r1389670316

Reply:
Got it - updated to `/user/{user}/developer/feed` in latest.